Core/Object Arts/Dolphin/MVP/Views/Common Controls/ListView.cls

```diff
@@ -1912,6 +1922,15 @@ selectionMarkIndex
 
 	^(self sendMessage: LVM_GETSELECTIONMARK) + 1.!
 
+selectionMarkIndex: anInteger
```

Comment:
If so, I would think the caret (the item with the focus ring) should also be preserved, and it is not. I'll make another PR fixing this for both ListView and ListBox.